src/main/java/nl/Tech/It/Easy/Tech/It/Easy/exceptions/ExceptionController.java

```diff
@@ -0,0 +1,16 @@
+package nl.Tech.It.Easy.Tech.It.Easy.exceptions;
+
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class ExceptionController {
+
+    @ExceptionHandler(RecordNotFoundException.class)
+    public ResponseEntity<String> handleRecordNotFoundException(RecordNotFoundException ex) {
+        return ResponseEntity.status(404).body(ex.getMessage());
```

Comment:
Je maakt goed gebruik van de builder. Als alternatief had je ook `ResponseEntity.notFound().body(message)` kunnen gebruiken. 

Je returned hier "ex.getMessage()", maar daar staat meestal te veel informatie in die je niet allemaal naar de gebruiker wilt sturen. Zoals je stacktrace, welke klassen jou uapplicatie bevat. welk framework je gebruikt. Daar is een hacker heel blij mee. Beter kun je je eigen message returnen (dat kan zelf dynamisch, afhankelijk van de waarde van ex.getMessage())